ElectronicObserver/Window/Dialog/QuestTrackerManager/Enums/GimmickType.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace ElectronicObserver.Window.Dialog.QuestTrackerManager.Enums;
+
+public enum GimmickType : int
```

Comment:
I think int is the default here anyway, so there's no need to write it explicitly.